soot-infoflow-android/src/soot/jimple/infoflow/android/source/AndroidSourceSinkManager.java

```diff
@@ -906,6 +906,32 @@ public void setAppPackageName(String appPackageName) {
 		this.appPackageName = appPackageName;
 	}
 
+	private SootMethod grabMethod(String sootClassName,String subSignature)
```

Comment:
Please add JavaDoc to your methods and make sure they have names that describe their purpose. In this case, the method should rather be named "grabMethodWithoutReturn" or something similar.

Reply:
Done